roles/common/tasks/main.yml

```diff
@@ -49,6 +58,7 @@
     - jq
     - java-1.8.0-openjdk
     - chrony
+    - smlr
```

Comment:
I put this in the common role b/c I think we'll be able to use it in other services (marathon waiting for zookeeper, for example) in future PRs. Let me know if anyone thinks this is a bad idea.